static/app/views/organizationContainer.tsx

```diff
@@ -35,7 +35,14 @@ function OrganizationContainer({children}: Props) {
   // organization in context**.
   //
   // TODO(epurkhiser): This scenario desprately should be improved
```

Comment:
Can you update the comment to reflect that this is now also used for there being no organizations.